src/diffusers/models/attention_processor.py

```diff
@@ -191,7 +191,10 @@ def set_use_memory_efficient_attention_xformers(
             elif hasattr(F, "scaled_dot_product_attention") and self.scale_qk:
                 warnings.warn(
                     "You have specified using flash attention using xFormers but you have PyTorch 2.0 already installed. "
-                    "We will default to PyTorch's native efficient flash attention implementation provided by PyTorch 2.0."
+                    "We will default to PyTorch's native efficient flash attention implementation (`F.scaled_dot_product_attention`) "
+                    "introduced in PyTorch 2.0. In case you are using LoRA or Custom Diffusion, we will fall "
+                    "back to their respective attention processors i.e., we will NOT use the PyTorch 2.0 "
+                    "native efficient flash attention."
```

Comment:
Would it be too much to say that you can use xFormers with LoRA?

Reply:
@pcuenca does the following sound good?

> "You have specified using flash attention using xFormers but you have PyTorch 2.0 already installed. "
"We will default to PyTorch's native efficient flash attention implementation (`F.scaled_dot_product_attention`) "
"introduced in PyTorch 2.0. In case you are using LoRA or Custom Diffusion, we will fall "
"back to their respective attention processors i.e., we will NOT use the PyTorch 2.0 "
"native efficient flash attention. In this case, addtionally, if you enable efficient attention via xformers "
"we will support that too."

Reply:
It sounds a bit confusing because in the first line we essentially say we are ignoring xFormers. Let's keep your original version, I think :) Sorry!

Reply:
@patrickvonplaten WDYT? I really want to ensure the users don't feel confused here.